.gitignore

```diff
@@ -0,0 +1,62 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
lots of files in this .gitignore are not related to project but to concrete user